app/controllers/users_controller.rb

```diff
@@ -15,7 +15,7 @@ def show
   def update
     if @user.update(user_params)
       render :show
-    else @user.errors
+    elsif @user.errors
       render json: @user.errors, status: :unprocessable_entity
     end
```

Comment:
Could there be a case where the update is unsuccessful and yet no error? If so, then we should have an `else` to handle that case. If not, then I think we should just change the `elsif @user.errors` to `else`.